Build issue on Windows

```
> cargo install    --git https://github.com/Macchina-CLI/macchina
...
   Compiling macchina v6.2.0 (C:\Users\ךינשגכהד\.cargo\git\checkouts\macchina-2bd55acfef0fa051\95b9c6d)
   Compiling wmi v0.12.2
error[E0308]: mismatched types
  --> src\theme\base.rs:14:24
   |
14 | #[derive(Debug, Clone, Serialize, Deserialize)]
   |                        ^^^^^^^^^
   |                        |
   |                        expected `Color`, found a different `Color`
   |                        arguments to this function are incorrect
   |
   = note: `Color` and `Color` have similar names, but are actually distinct types
note: `Color` is defined in crate `ratatui`
  --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ratatui-0.26.3\src\style\color.rs:66:1
   |
66 | pub enum Color {
   | ^^^^^^^^^^^^^^
note: `Color` is defined in crate `ratatui`
  --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ratatui-0.27.0\src\style\color.rs:66:1
   |
66 | pub enum Color {
   | ^^^^^^^^^^^^^^
   = note: perhaps two different versions of crate `ratatui` are being used?
note: function defined here
  --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\color-to-tui-0.3.0\src\lib.rs:7:8
   |
7  | pub fn serialize<S: Serializer>(color: &Color, serializer: S) -> Result<...
   |        ^^^^^^^^^
   = note: this error originates in the derive macro `Serialize` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: `?` operator has incompatible types
  --> src\theme\base.rs:14:35
   |
14 | #[derive(Debug, Clone, Serialize, Deserialize)]
   |                                   ^^^^^^^^^^^ expected `Color`, found a different `Color`
   |
   = note: `?` operator cannot convert from `ratatui::style::color::Color` to `ratatui::style::Color`
   = note: `Color` and `Color` have similar names, but are actually distinct types
note: `Color` is defined in crate `ratatui`
  --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ratatui-0.27.0\src\style\color.rs:66:1
   |
66 | pub enum Color {
   | ^^^^^^^^^^^^^^
note: `Color` is defined in crate `ratatui`
  --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ratatui-0.26.3\src\style\color.rs:66:1
   |
66 | pub enum Color {
   | ^^^^^^^^^^^^^^
   = note: perhaps two different versions of crate `ratatui` are being used?
   = note: this error originates in the derive macro `Deserialize` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: mismatched types
   --> src\theme\components.rs:116:33
    |
116 | #[derive(Debug, Default, Clone, Serialize, Deserialize)]
    |                                 ^^^^^^^^^
    |                                 |
    |                                 expected `Color`, found a different `Color`
    |                                 arguments to this function are incorrect
    |
    = note: `Color` and `Color` have similar names, but are actually distinct types
note: `Color` is defined in crate `ratatui`
   --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ratatui-0.26.3\src\style\color.rs:66:1
    |
66  | pub enum Color {
    | ^^^^^^^^^^^^^^
note: `Color` is defined in crate `ratatui`
   --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ratatui-0.27.0\src\style\color.rs:66:1
    |
66  | pub enum Color {
    | ^^^^^^^^^^^^^^
    = note: perhaps two different versions of crate `ratatui` are being used?
note: function defined here
   --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\color-to-tui-0.3.0\src\optional.rs:4:8
    |
4   | pub fn serialize<S: Serializer>(color: &Option<Color>, serializer: S) -...
    |        ^^^^^^^^^
    = note: this error originates in the derive macro `Serialize` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0308]: `?` operator has incompatible types
   --> src\theme\components.rs:116:44
    |
116 | #[derive(Debug, Default, Clone, Serialize, Deserialize)]
    |                                            ^^^^^^^^^^^ expected `Color`, found a different `Color`
    |
    = note: `?` operator cannot convert from `std::option::Option<ratatui::style::color::Color>` to `std::option::Option<ratatui::style::Color>`
    = note: `Color` and `Color` have similar names, but are actually distinct types
note: `Color` is defined in crate `ratatui`
   --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ratatui-0.27.0\src\style\color.rs:66:1
    |
66  | pub enum Color {
    | ^^^^^^^^^^^^^^
note: `Color` is defined in crate `ratatui`
   --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ratatui-0.26.3\src\style\color.rs:66:1
    |
66  | pub enum Color {
    | ^^^^^^^^^^^^^^
    = note: perhaps two different versions of crate `ratatui` are being used?
    = note: this error originates in the derive macro `Deserialize` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0308`.
error: could not compile `macchina` (bin "macchina") due to 9 previous errors
error: failed to compile `macchina v6.2.0 (https://github.com/Macchina-CLI/macchina#95b9c6df)`, intermediate artifacts can be found at `C:\Users\0BAC~1\AppData\Local\Temp\cargo-installDh2ZZd`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

Adding `--locked` yields a different error
```
error[E0282]: type annotations needed for `Box<_>`
  --> C:\Users\ךינשגכהד\.cargo\registry\src\index.crates.io-6f17d22bba15001f\time-0.3.29\src\format_description\parse\mod.rs:83:9
   |
83 |     let items = format_items
   |         ^^^^^
...
86 |     Ok(items.into())
   |              ---- type must be known at this point
   |
help: consider giving `items` an explicit type, where the placeholders `_` are specified
   |
83 |     let items: Box<_> = format_items
   |              ++++++++

For more information about this error, try `rustc --explain E0282`.
error: could not compile `time` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `macchina v6.2.0 (https://github.com/Macchina-CLI/macchina#95b9c6df)`, intermediate artifacts can be found at `C:\Users\0BAC~1\AppData\Local\Temp\cargo-installbE2Xc4`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,13 +21,17 @@ rand = "0.8.5"
 unicode-width = "0.1.11"
 lazy_static = "1.4.0"
 ansi-to-tui = "4.0.1"
-color-to-tui = "0.3.0"
 dirs = "5.0.1"
 toml = { version = "0.8.8", features = ["parse"] }
 serde_json = "1.0.107"
 thiserror = "1.0.49"
 ratatui = { version = "0.26", default-features = false, features = ["crossterm"] }
 serde = { version = "1.0.188", features = ["derive"] }
+
+[dependencies.color-to-tui]
+version = "0.3.0"
+ratatui = { version = "0.26", default-features = false, features = ["crossterm"] }
+
 
 [build-dependencies.vergen]
 version = "8.2.6"

--- a/src/bars.rs
+++ b/src/bars.rs
@@ -5,7 +5,7 @@ in a bar.
 
 For example:
 - __CPU Usage__ ranges from 0 to 100%, this function can return a `usize`
-that tells the bar how many of its blocks should be represented as being used.
+  that tells the bar how many of its blocks should be represented as being used.
 
 The same goes for __battery percentage__, as it ranges from 0 to 100%.
 */

--- a/src/format.rs
+++ b/src/format.rs
@@ -134,7 +134,7 @@ pub fn packages(packages: Vec<(PackageManager, usize)>) -> Result<String, Readou
 
     for (i, (pm, count)) in packages.iter().enumerate() {
         let add_comma = if i + 1 < len { ", " } else { "" };
-        string.push_str(&format!("{} ({}){}", count, pm.to_string(), add_comma));
+        string.push_str(&format!("{} ({}){}", count, pm, add_comma));
     }
 
     Ok(string)

